projects/frontend-ui/shared-components/README.md

```diff
@@ -0,0 +1,15 @@
+# VDK Shared Components library
+
+This project implements common UI functionality that is used in the data-pipelines project.
+But these components are also used by private projects within VMWare.
```

Comment:
I would rephrase to emphasise these components are agnostic (non data-pipelines UI -specific), and could be used for extension with other GUI components

Reply:
Done